src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -43,6 +45,7 @@ export class ErrorInfo {
 export class NotebookModel extends Disposable implements INotebookModel {
 	private _contextsChangedEmitter = new Emitter<void>();
 	private _contentChangedEmitter = new Emitter<NotebookContentChange>();
+	private _modelLoadedEmitter = new Emitter<boolean>();
```

Comment:
This is never hooked up to, and the event never fires. Looks like it's left over from something, or did you just miss hooking it up and really still need to add in more code? Either way, please change? Thanks.

Reply:
Raj - I'm not seeing this being fixed either by emitting the event or removing this. What am I missing?